usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -49,7 +49,23 @@ def get_award_data(db_cursor, award_type, max_id, ids=None):
         return dictfetchall(db_cursor)
 
     @staticmethod
-    def get_award(row, award_type):
+    def generate_unique_ids(row, award_type):
+        if award_type == 'procurement':
+            # "CONT_AW_" + agency_id + referenced_idv_agency_iden + piid + parent_award_id
+            # "CONT_AW_" + contract_agency_code + contract_idv_agency_code + contract_number + idv_reference_number
+            return \
+                'CONT_AW_' + \
+                (row['contract_agency_code'].replace('-', '') if row['contract_agency_code'] else '-NONE-') + \
+                '_' + \
+                (row['contract_idv_agency_code'].replace('-', '') if row['contract_idv_agency_code'] else '-NONE-') + \
+                '_' + \
+                (row['contract_number'].replace('-', '') if row['contract_number'] else '-NONE-') + \
+                '_' + \
+                (row['idv_reference_number'].replace('-', '') if row['idv_reference_number'] else '-NONE-')
+        else:
+            return row['fain'].replace('-', '')
```

Comment:
Is this correct? Doesn't appear to be creating a `CONT_AW_...` string

Reply:
Added comments. Assistance award identifiers typically are prepended with `ASST_AW_` but we don't have all the necessary fields to actually generate it, so we can only go off of the `fain`.